responses/_recorder.py

```diff
@@ -64,6 +75,7 @@ def _dump(
     registered: "List[BaseResponse]",
     destination: "Union[BinaryIO, TextIOWrapper]",
     dumper: "Callable[[Union[Dict[Any, Any], List[Any]], Union[BinaryIO, TextIOWrapper]], Any]",
+    additional_headers: "Optional[List[str]]" = None,
```

Comment:
Should this be `include_headers` or `capture_headers`? On first read my interpretation was that these headers would be added into the recorded responses.